manifests/params.pp

```diff
@@ -12,6 +12,7 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
+  $postgres_datadir          = '/var/lib/postgresql/9.4/main'
```

Comment:
We definitely should not be reflecting our own opinions for the default of this parameter over in the puppetdb module, those opinions should be expressed in postgresql only. Otherwise, we're just going to play a game of out-of-sync catch up over time, if upstream changes its values for example.